.CI/build_all

```diff
@@ -24,7 +24,7 @@ echo ""
 echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
-conda install --yes --quiet conda-build-all=1.0.0
```

Comment:
We need latest `conda-build-all` for `R` packages. I am not sure that pinning is still needed anyways.